File: 0001 Code Chef/0004 Ides of march/implementation/0005 kotlin/IdesOfMarch.kt
fun main(args:Array<String>){
	var stringInput:String? = readLine();
	var day:Int? = stringInput?.toIntOrNull();
	if(day != null){
		if(day == 15){
			println("YES");
		}else{
			println("NO");
		}
	}	
}

File: 0001 Code Chef/0004 Ides of march/implementation/0003 c#/IdesOfMarch.cs
using System;

class IdesOfMarch{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int day = Convert.ToInt32(stringInput);
        if(day == 15){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}